src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -731,8 +735,31 @@ private List<EquipmentModificationInfos> execModifyLoad(NetworkStoreListener lis
                 .collect(Collectors.toList());
     }
 
+    private static <T> void applyElementaryModifications(Consumer<T> setter, Supplier<T> getter, AttributeModification<T> modification) {
+        if (modification != null) {
+            setter.accept(modification.applyModification(getter.get()));
+        }
+    }
+
+    private void applyModifications(Network network, GeneratorModificationInfos modificationInfos) {
```

Comment:
Maybe rename to applyGeneratorModifcations to avoid confusion with further equipment modification methods.

Reply:
no, i want to apply a modification, which one is not my business, and it is easier to generify this way.

Reply:
Ok, why not